src/index.js

```diff
@@ -740,3 +742,7 @@ const SELF_CLOSING = new Set([
 export default renderToString;
 export const render = renderToString;
 export const renderToStaticMarkup = renderToString;
+
+function isSignal(x) {
+	return x instanceof Signal;
+}
```

Comment:
Wondering if we should use duck-typing here like https://github.com/denoland/fresh/blob/e65643c482de9b5afee5d52af3ccc88941c35b5c/src/runtime/server/preact_hooks.tsx#L355-L362 . That way we can keep rts dependency free and don't have to keep it in sync with major `@preact/signals` version increases.

Reply:
Not sure if `x !== null && typeof x === "object"` is more or less expensive than an `instanceof` check. I let you decide on that one.